client/client.js

```diff
@@ -0,0 +1,14 @@
+$(function () {
+var socket = io();
+var bruger = "Bob";
+$('#bruger').val(bruger);
+$('form').submit(function(){
+  socket.emit('chat message',{message: $('#m').val(), bruger: $('#bruger').val()});
+  $('#m').val('');
+  return false;
+});
+socket.on('chat message', function(msg){
+  var avatar = '<img src="https://api.adorable.io/avatars/15/' + msg.bruger + '.png" style="width: 15px; height: 15px; border-radius: 50%;">';
```

Comment:
`15` is repeated a few times here, would be great if you created a constant for it. It could be called `AVATAR_SIZE`. 

Then you can change it in one place.